### ISSUE TITLE
Cursor should support alive property

### ISSUE THREAD

**Issue by radifalco** [mongomock/mongomock#446]
This can be implemented pretty easily by adding this to the `Cursor` class.

```
    @property
    def alive(self):
        return self._emitted != self.count()
```

**Comment by pcorpet** [comment]
Hi @radifalco , thanks for the report. We welcome Pull Requests :-)

**Comment by pcorpet** [comment]
Implemented in 82317dd, thanks @radifalco for the idea and the code.
